src/syncerd/bitcoin_syncer.rs

```diff
@@ -759,6 +767,30 @@ fn height_polling(
 
                 // if the blocks changed, query transactions
```

Comment:
...and check all pending broadcast tx

Reply:
1a4de0ad60db310e4e40c20dec2fd05cf9e088f0